data.sql

```diff
@@ -91,3 +91,248 @@ UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Dean Winchester')
   WHERE name IN ('Angemon', 'Boarmon');
 
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Agumon', '02-03-2020', 0, true, 10.23);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Gabumon', '11-15-2018', 2, true, 8);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Pikachu', '01-07-2021', 1, false, 15.04);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Devimon', '05-12-2017', 5, true, 11);
+
+
```

Comment:
**[Optional]**

- [ ] Your readme is not the latest one from microverse template :fearful: Kindly use the latest version of the readme template :+1: using this [link](https://github.com/microverseinc/readme-template/blob/master/README.md) :blush: